### PR TITLE
created card hover effcet files

### PR DESCRIPTION
**Pull request by Rehan-Khan-Engineer**

### CODE REVIEW

File: tasks-by-users/Rehan-Khan-Engineer/04-card-hover-effect/style.css [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  background: #ff005f;
+
+  transition: 0.5s;
+  transform: translateY(0);
+}
+
+.row-1 h3 {
+  margin: 0;
+  margin-left: 40%;
+  margin-bottom: 10%;
```
Comment (Abbas4tech) [inline tasks-by-users/Rehan-Khan-Engineer/04-card-hover-effect/style.css:59]:
````css
margin: 0 0 10% 40%;
````

File: tasks-by-users/Rehan-Khan-Engineer/04-card-hover-effect/index.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+            quo, recusandae ab similique deserunt. At magnam esse ipsum neque
+            quasi.
+          </p>
+          <a>Read More</a>
+        </div>
+      </div>
+      <div class="row">
+        <div class="row-1">
+          <img
+            alt="Image"
```
Comment (Abbas4tech) [inline tasks-by-users/Rehan-Khan-Engineer/04-card-hover-effect/index.html:35]: alt must describe about image if it is not rendered! 
what does `alt="Image"` describe ?

**Review by Abbas4tech — CHANGES_REQUESTED** [review]
Correct alt for images and use shorthand properties

File: tasks-by-users/Rehan-Khan-Engineer/04-card-hover-effect/style.css [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+  transform: translateY(100px);
+
+  z-index: 1;
+
+  background: #333;
+}
+
+.row:hover .row-1 {
+  background: #ff005f;
```
Comment (mmusk) [inline tasks-by-users/Rehan-Khan-Engineer/04-card-hover-effect/style.css:50]: use `backkground-color` instead

File: tasks-by-users/Rehan-Khan-Engineer/04-card-hover-effect/style.css (hunk 1 above)
Comment (Rehan-Khan-Engineer) [reply tasks-by-users/Rehan-Khan-Engineer/04-card-hover-effect/style.css:59]: Done @Abbas4tech

File: tasks-by-users/Rehan-Khan-Engineer/04-card-hover-effect/index.html (hunk 2 above)
Comment (Rehan-Khan-Engineer) [reply tasks-by-users/Rehan-Khan-Engineer/04-card-hover-effect/index.html:35]: Done @Abbas4tech

File: tasks-by-users/Rehan-Khan-Engineer/04-card-hover-effect/style.css (hunk 3 above)
Comment (Rehan-Khan-Engineer) [reply tasks-by-users/Rehan-Khan-Engineer/04-card-hover-effect/style.css:50]: done @mmusk

**Comment by Rehan-Khan-Engineer** [comment]
> Correct alt for images and use shorthand properties

Done @Abbas4tech

**Review by shaikh-7abish — APPROVED** [review]

**Review by Abbas4tech — APPROVED** [review]

File: tasks-by-users/Rehan-Khan-Engineer/04-card-hover-effect/style.css [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+  margin-top: 4%;
+
+  opacity: 0.2;
+
+  text-align: center;
+}
+
+.row-2 {
+  height: 200px;
+  width: 300px;
```
Comment (nasir-k) [inline tasks-by-users/Rehan-Khan-Engineer/04-card-hover-effect/style.css:83]: You should use grouping selector to group similar properties.

**Review by nasir-k — CHANGES_REQUESTED** [review]

File: tasks-by-users/Rehan-Khan-Engineer/04-card-hover-effect/style.css (hunk 4 above)
Comment (Rehan-Khan-Engineer) [reply tasks-by-users/Rehan-Khan-Engineer/04-card-hover-effect/style.css:83]: ok @nasir-k

**Review by nehalskdev — APPROVED** [review]

**Review by mmusk — APPROVED** [review]

**Review by nasir-k — APPROVED** [review]

**Pull request merged**